resources/functionMap.php

```diff
@@ -10373,11 +10373,11 @@
 'session_destroy' => ['bool'],
 'session_encode' => ['string|false'],
 'session_gc' => ['int|false'],
-'session_get_cookie_params' => ['array'],
+'session_get_cookie_params' => ['array{lifetime:0|positive-int,path:non-falsy-string,domain:string,secure:bool,httponly:bool,samesite:string}'],
 'session_id' => ['string|false', 'newid='=>'string'],
 'session_is_registered' => ['bool', 'name'=>'string'],
 'session_module_name' => ['string|false', 'newname='=>'string'],
-'session_name' => ['string|false', 'newname='=>'string'],
+'session_name' => ['non-falsy-string|false', 'newname='=>'string'],
```

Comment:
Both `'0'` and `''` are not valid session names.

https://github.com/php/php-src/blob/ccdc21b39a286e0d90f6f392ac54b6f8d465d518/ext/session/session.c#L674-L687